benchmarks/allocation/alloc.cc

```diff
@@ -34,7 +35,10 @@ void __cheri_compartment("allocbench") run()
 		auto quota = heap_quota_remaining(MALLOC_CAPABILITY);
 		Debug::Invariant(quota == MALLOC_QUOTA, "Quota remaining {}, should be {}", quota, MALLOC_QUOTA);
 		Debug::log("Flushing quarantine");
-		heap_quarantine_empty();
+		Debug::Invariant(heap_quarantine_empty() >= 0,
```

Comment:
I think in _benchmark_ code, these might want to be `Debug::Assert`, so that they compile away in release mode when we're measuring cycles?

Reply:
This doesn't work out super well, because the call won't evaluated when the assert is compiled out.